filebeat/docs/multiple-prospectors.asciidoc

```diff
@@ -18,8 +23,11 @@ filebeat:
     -
       paths:
         - "/var/log/apache2/*"
+      include_lines: ["apache"]
+      exclude_lines: ["^DBG"]
```

Comment:
one nice use-case I've seen was:

```
fields:
  json: true
fields_under_root: true
```

Then in logstash you can filter and do json parsing like:

```
if [json] {
}
```